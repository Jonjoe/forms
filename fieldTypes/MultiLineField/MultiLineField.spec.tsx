import React from "react";
import renderer from "react-test-renderer";

import Example from "./MultiLineField.example";

test("MultiLineField renders correctly", () => {
  const component = renderer.create(<Example />);

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
