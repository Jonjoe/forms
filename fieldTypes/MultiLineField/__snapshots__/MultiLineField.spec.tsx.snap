// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MultiLineField renders correctly 1`] = `
<textarea
  className="MultiLineField  "
  onChange={[Function]}
  value=""
/>
`;
